hedera-node/src/main/java/com/hedera/services/files/interceptors/ThrottleDefsManager.java

```diff
@@ -121,7 +121,8 @@ public OptionalInt priorityForCandidate(FileID id) {
 	public void postUpdate(FileID id, byte[] contents) {
 		/* Note - here we trust the file system to correctly invoke this interceptor
 		only when we returned a priority from {@code priorityForCandidate}. */
-		postUpdateCb.accept(uncheckedParseFrom(contents).get());
+		final Optional<ThrottleDefinitions> throttleDefinitions = uncheckedParseFrom(contents);
+		throttleDefinitions.ifPresent(postUpdateCb);
```

Comment:
I think it should be `throttleDefinitions.ifPresent(postUpdateCb.accept(throttleDefinitions));`

Reply:
it would automatically apply as we are passing a consumer.